src/main/resources/assets/commandblockide/lang/de_de.json

```diff
@@ -0,0 +1,23 @@
+{
+    "arguments.editfunction.tag.unsupported": "Kann keine Funktions Tags bearbeiten",
+    "commands.editfunction.failed.modNotInstalled": "Die Command Block IDE muss installiert sein",
+    "commandBlockIDE.close": "Schließen",
+    "commandBlockIDE.save": "Speichern",
+    "commandBlockIDE.type.redstone": "Impuls",
+    "commandBlockIDE.type.redstoneKonditional": "Impuls, Konditional",
+    "commandBlockIDE.type.auto": "Wiederhohlen",
+    "commandBlockIDE.type.autoKonditional": "Wiederhohlen, Konditional",
+    "commandBlockIDE.type.sequence": "Verketten",
+    "commandBlockIDE.type.sequenceKonditional": "Verketten, Konditional",
+    "commandBlockIDE.unloaded": "Ungeladen",
+    "commandBlockIDE.lastOutput.on": "Sichere Ausgabe",
+    "commandBlockIDE.lastOutput.off": "Keine Ausgabe",
+    "commandBlockIDE.lastOutput.none": "Keine Ausgabe",
```

Comment:
Why are these two the same?

Reply:
It could/should be:
line 13: `Verfolgung der Ausgabe`
line 14: `Keine Verfolgung der Ausgabe`
line 15: `Keine Ausgabe`
But wouldn't this be a bit too long? 🤔